pro/src/screens/SignupJourneyForm/Offerers/__specs__/Offerers.spec.tsx

```diff
@@ -281,4 +283,74 @@ describe('screens:SignupJourney::Offerers', () => {
     renderOfferersScreen(contextValue)
     expect(await screen.findByText('Offerer screen')).toBeInTheDocument()
   })
+
+  describe('modal handling', () => {
+    it('should render component modal', async () => {
+      renderOfferersScreen(contextValue)
+
+      await userEvent.click(await screen.findByText('Rejoindre cet espace'))
+
+      expect(
+        await screen.findByText(
+          'Êtes-vous sûr de vouloir rejoindre cet espace PRO ?'
+        )
+      ).toBeInTheDocument()
+    })
+
+    it('should close modal on cancel', async () => {
+      renderOfferersScreen(contextValue)
+
+      await userEvent.click(await screen.findByText('Rejoindre cet espace'))
+
+      expect(
+        await screen.findByText(
+          'Êtes-vous sûr de vouloir rejoindre cet espace PRO ?'
+        )
+      ).toBeInTheDocument()
+
+      await userEvent.click(await screen.findByText('Annuler'))
+
+      expect(
+        screen.queryByText(
+          'Êtes-vous sûr de vouloir rejoindre cet espace PRO ?'
+        )
+      ).not.toBeInTheDocument()
+    })
+
+    it('should submit form', async () => {
+      renderOfferersScreen(contextValue)
+      jest.spyOn(api, 'createOfferer').mockResolvedValue(expect.anything())
+      jest.spyOn(getSirenDataAdapter, 'default').mockResolvedValue({
+        isOk: true,
+        message: '',
+        payload: {
+          values: {
+            address: '',
+            city: 'lille',
+            name: '',
+            postalCode: '59000',
+            siren: '',
+            apeCode: '',
+          },
+        },
+      })
+
+      await userEvent.click(await screen.findByText('Rejoindre cet espace'))
+
+      expect(
+        await screen.findByText(
+          'Êtes-vous sûr de vouloir rejoindre cet espace PRO ?'
+        )
+      ).toBeInTheDocument()
+
+      await userEvent.click(await screen.findByText('Rejoindre cet espace PRO'))
+
```

Comment:
On teste aussi que la pop in est fermée ?

Reply:
Je vais plutôt rajouter le test de la redirection